appcues/src/main/java/com/appcues/trait/StickyContentPadding.kt

```diff
@@ -6,37 +6,69 @@ import androidx.compose.runtime.derivedStateOf
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.ui.unit.Density
 
+/**
+ * Type that defines the padding values necessary to account for sticky
+ * content elements in a composition.
```

Comment:
not sure if we can be more specific with something like:

Type that defines the padding values necessary to account for sticky content elements when composing the step

Reply:
yeah that sounds good to me, updated, thanks